app/helpers/map_locations_helper.rb

```diff
@@ -64,7 +64,27 @@ def prepare_map_settings(map_location, editable, parent_class, investments_coord
     }
     options[:marker_latitude] = map_location.latitude if map_location.latitude.present?
     options[:marker_longitude] = map_location.longitude if map_location.longitude.present?
+
+    if render_related_proposals?(map_location, editable)
+      related_proposals_coordinates = map_related_proposals_coordinates(map_location.proposal)
+
+      if related_proposals_coordinates.present?
+        options[:marker_related_proposals_coordinates] = related_proposals_coordinates
+        options[:related_proposals_layer_text] = t("proposals.show.related_proposals_layer")
```

Comment:
```suggestion
        options[:related_proposals_layer_text] = t("map.related_proposals")
```
Or:

```suggestion
        options[:related_proposals_layer_text] = t("map.proposals.related")
```

We could add the texts to either `config/locales/custom/*` or `config/locales/**/general.yml`. Both of them have a `map` entry.